src/transformers/modeling_utils.py

```diff
@@ -1876,29 +1883,29 @@ def get_memory_footprint(self, return_buffers=True):
 
     def to(self, *args, **kwargs):
         # Checks if the model has been loaded in 8-bit
-        if getattr(self, "is_loaded_in_8bit", False):
+        if getattr(self, "is_loaded_in_kbit", False):
             raise ValueError(
-                "`.to` is not supported for `8-bit` models. Please use the model as it is, since the"
+                "`.to` is not supported for `k-bit` models. Please use the model as it is, since the"
```

Comment:
```suggestion
                "`.to` is not supported for `4-bit` or `8-bit` models. Please use the model as it is, since the"
```
The end user won't know what `k-bit` stands for.